mentat/llm_api.py

```diff
@@ -82,11 +82,15 @@ async def warn_user(message: str, max_tries: int, details: Details):
     await stream.send(warning, color="light_yellow")
 
 
+# This can be mocked in benchmarks to increase the wait time
+backoff_delay_base = 2
```

Comment:
does this work? I don't know when python decorator args get applied

Reply:
I think they'd get applied at import time, as the decorator would run then to create the decorated functions

Reply:
Good point, I'm not entirely sure. I'll test it and if it doesn't work I'll remove this

Reply:
It didn't work; we could probably increase the delay only in benchmarks by checking if PYTEST is active, but I'm honestly not sure we need to (if the openai api is down for 40 seconds, it's probably down for a while); I'm just going to merge and we can figure this out later if we do need to.